test/forkMainnet/SmartOrderStrategy/Setup.t.sol

```diff
@@ -40,6 +40,11 @@ contract SmartOrderStrategyTest is Test, Tokens, BalanceUtil {
             setERC20Balance(tokenList[i], genericSwap, 10000);
         }
 
+        deal(USDC_ADDRESS, address(smartOrderStrategy), 1 wei);
+        deal(USDT_ADDRESS, address(smartOrderStrategy), 1 wei);
+        deal(WETH_ADDRESS, address(smartOrderStrategy), 1 wei);
+        deal(WBTC_ADDRESS, address(smartOrderStrategy), 1 wei);
```

Comment:
Suggestion: Try testing if leaving one wei works on first swap instead  of dealing to contract directly.

Reply:
Same with David.

Reply:
I prefer sticking with the current approach for these tests, and instead, introducing new test scenarios to verify that leaving 1 wei works on the first swap. Any thoughts on that?